source/_integrations/plugwise.markdown

```diff
@@ -236,33 +240,49 @@ script:
           preset_mode: asleep
 ```
 
-### Supported devices
+## Supported devices
+
+### Adam
+
+A complete zone control system also known as Adam HA, supporting:
+
+- On/Off, OpenTherm or Loria/Thermastage heating, and cooling support.
```

Comment:
Maybe take a bit more info from here: https://www.plugwise.com/product/adam-ha/?lang=en

Reply:
Reworked new comments by just adding the links to the website (forgoing the need to be fully copy/pasting). As a 'supported devices' section I'd assume it's a mere list of things supported. If we are missing things it should go in the top description I'd suppose (for that is what the user will mostly read if interested, supported devices is more when you want to make sure things you buy/bought are going to be supported?)

Reply:
Could you check if the intro and links on top would be sufficient?